src/api/courses.js

```diff
@@ -69,6 +69,124 @@ router.get(
   })
 )
 
+// Get course queue data
+router.get(
+  '/:courseId/data',
```

Comment:
I would consider scoping this to something like `data/questions` or the like - we might want more "data" endpoints in the future. One that immediately comes to mind is a `data/staff` endpoint that could return all join/leave records of queue staff.